mesmer/mesmer_m/harmonic_model.py

```diff
@@ -34,7 +32,7 @@ def generate_fourier_series_np(coeffs, order, yearly_T, months):
         Fourier Series of order n calculated over yearly_T and months with coeffs.
 
     """
-    # TODO: infer order from coeffs
+    order = int(len(coeffs) / 4)
```

Comment:
```suggestion
    order = int(coeffs.size / 4)
```
if it's guaranteed to be a 1D array. (Could also do `// 4` but `int` is more explicit.)

Reply:
Hm when we make the function private it should be guaranteed that it's 1D. Or do you prefer to implement a check?

Reply:
I quickly checked and `len(arr) == arr.shape[0]` - so the assumption is there already, and I also suspect it would fail below. Thus, I suggest we go ahead and commit this suggestion.